index.md

```diff
@@ -1,14 +1,23 @@
 ---
 sidebar: false
-title: FAIRDOM-SEEK Documentation
+title: FAIRDOM-SEEK Documentation # Replace with instance name
 ---
+{% assign seek_instance_name = site.seek_instance.name | default: "FAIRDOM-SEEK" %}
+
+<!--Complete the description of your own instance here or delete it.-->
+<div>
+{% unless seek_instance_name == "FAIRDOM-SEEK" %}
+    ## About {{ seek_instance_name }}
+    The {{ seek_instance_name }} platform is based on the FAIRDOM-SEEK software.
```

Comment:
```suggestion
    The {{ seek_instance_name }} platform is based on the [FAIRDOM-SEEK](https://seek4science.org/) software.
```
Suggest you link to the main FAIRDOM-SEEK website here. Also, could you please show me how this div looks on DataHub? Perhaps we could style this differently, or use an ETT info box?